src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -274,6 +327,9 @@ def __call__(
             image = self.numpy_to_pil(image)
 
         if not return_dict:
-            return (image, has_nsfw_concept)
+            return (image, has_nsfw_concept, all_latents)
```

Comment:
I think if PipelineOutput classes are the way forward and the tuple return format here is mainly for backwards compatibility, we should leave it the same size it was (a pair) and not worry about adding new features to it.

Reply:
Since this method had a `@torch.no_grad` decorator, I don't think this is for backwards compatibility 🤔 
But looking at the transformers package, it seems that[ when `return_dict_generate=False`, options like `output_scores`/`output_attentions` don't matter](https://github.com/huggingface/transformers/blob/main/src/transformers/generation_utils.py#L2078), so it makes sense to remove `latents` from the tuple as you mention :)